dnd5esheets/front/package.json

```diff
@@ -6,7 +6,7 @@
     "start": "solid-start start",
     "generate-client": "openapi --input ./openapi.json --output ./src/5esheets-client",
     "generate-effect-parser": "peggy --plugin ./node_modules/ts-pegjs/dist/tspegjs -o src/effects/parser/index.ts src/effects/parser/grammar.pegjs",
-    "test": "vitest --run",
+    "test": "vitest",
```

Comment:
This seems to work on CI

Reply:
Not sure what you mean here. In doubt, here is the rationale:
`vite` will automatically detect if run in CI or not, and will use the appropriate behavior:
- in CI: `--run` (run once, without watching)
- locally: watch, and run affected tests at every file change

Reply:
I meant that because it watches files locally and reruns the test, I expected it to do the same in CI. But actually, as you pointed out, it works well in CI as well 👍